src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -451,4 +451,16 @@ void tiebreakPreviousRoundCountsThenRandomTest() {
   void treatBlankAsUndeclaredWriteInTest() {
     runTabulationTest("test_set_treat_blank_as_undeclared_write_in");
   }
+
+  @Test
+  @DisplayName("undeclared write-in (UWI) test")
+  void uwiTest() {
+    runTabulationTest("uwi_test");
```

Comment:
Feel free to veto this if it doesn't make sense, but could we get a more descriptive name for this, since we have a couple other UWI tests? If it's a pain to change the file names, even just the DisplayName would probably help.